### PR TITLE
connected-database

### PR DESCRIPTION
**Pull request by Tiago-Varela4**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -42,6 +59,16 @@
             . '</div> ';
     }
     echo $html;
+
+    $query = $pdo->prepare('SELECT `shows`.`name`, `genres`.`name` AS `genre`, `release year` FROM `shows` INNER JOIN `genres` ON `shows`.`genre` = `genres`.`id`;');
+
+    $query->execute();
+
+    $result = $query->fetchALL();
```
Comment (claudia-lim) [inline index.php:67]: Is $result meant to be $items?

Comment (Tiago-Varela4) [reply index.php:67]: yeah i was testing to see if i could get the database items to show up

Comment (Tiago-Varela4) [reply index.php:67]: echo '<pre>';
    print_r($result);
    echo '</pre>';
    ?>

Comment (claudia-lim) [reply index.php:67]: Okay sounds good, approved!

**Review by claudia-lim — APPROVED** [review]

**Pull request merged**